### PR TITLE
PolEthBridge:  Update bridge owner and README as well as add missing natspec

### PR DESCRIPTION
**Pull request by efecarranza**
# Changelog

Update deployment script with correct owner.
Add missing natspec of Bridge contract.
Update README with information on how to process multiple burns in a single transaction.

### CODE REVIEW

**Comment by sendra** [comment]
I think, now that its being updated. It would be better to move events, natspec, and so on to the interface, and then use @inheritdocs.

**Comment by efecarranza** [comment]
@sendra updated!

File: src/bridges/AavePolEthERC20Bridge.sol [hunk 1]
```diff
@@ -34,7 +34,7 @@ contract AavePolEthERC20Bridge is Ownable, Rescuable, IAavePolEthERC20Bridge {
 
   error InvalidChain();
 
-  event Exit();
+  event Exit(bytes proof);
   event Bridge(address token, uint256 amount);
   event WithdrawToCollector(address token, uint256 amount);
```
Comment (sendra) [inline src/bridges/AavePolEthERC20Bridge.sol:40]: Missing natspec on interface

File: src/bridges/AavePolEthERC20Bridge.sol [hunk 2]
```diff
@@ -44,38 +44,34 @@ contract AavePolEthERC20Bridge is Ownable, Rescuable, IAavePolEthERC20Bridge {
     _transferOwnership(_owner);
```
Comment (sendra) [inline src/bridges/AavePolEthERC20Bridge.sol:48]: constructor missing natspec (on this file)

**Comment by sendra** [comment]
address public constant ROOT_CHAIN_MANAGER missing natspec on interface (as this code was not change, could not leave comment directly on code)

**Comment by efecarranza** [comment]
@sendra updated, thank you. i added a new function and update the README accordingly to add a convenience function to check whether a token is mapped prior to burning.

**Review by Zer0dot — APPROVED** [review]

**Comment by efecarranza** [comment]
@Zer0dot alright, i've added the relevant TXs and now it forward ETH as well (when withdrawing WETH)

**Review by sendra — APPROVED** [review]

**Review by Zer0dot — APPROVED** [review]

**Pull request merged**